core/server/proxy/src/main/java/alluxio/proxy/s3/S3RestUtils.java

```diff
@@ -210,10 +213,18 @@ public static String parsePath(String bucketPath) {
    *
    * @param exception Exception thrown when process s3 object rest request
    * @param resource complete bucket path
+   * @param auditContext the audit context for exception
    * @return instance of {@link S3Exception}
    */
-  public static S3Exception toBucketS3Exception(Exception exception, String resource) {
+  public static S3Exception toBucketS3Exception(Exception exception, String resource,
+                                                S3AuditContext auditContext) {
```

Comment:
Is this supposed to be `alluxio.master.audit.AuditContext`? I don't have a preference, just want them to be consistent.